scripts/vm/hypervisor/xenserver/vmops

```diff
@@ -1267,9 +1267,9 @@ def cache_ipset_keyword():
     type = getIpsetType()
     tmpname = 'ipsetqzvxtmp'
     try:
-        util.pread2(['/bin/bash', '-c', 'ipset -N ' + tmpname + type])
+        util.pread2(['/bin/bash', '-c', 'ipset', '-N ',  tmpname , type])
```

Comment:
Spaces will not cause any issues. Anyway I removed the spaces, tested and update the patch.